sample-app/src/components/VisualAutocomplete/SampleVisualSearchBar.tsx

```diff
@@ -0,0 +1,78 @@
+import VisualSearchBar from './VisualSearchBar';
+import { Result } from '@yext/answers-headless-react';
+import VisualEntities from './VisualEntities';
+
+/**
+ * This is an example of how to use the VisualSearchBar component.
+ */
+export default function SampleVisualSearchBar() {
+  return (
+    <VisualSearchBar
+      placeholder='Search...'
+      screenReaderInstructionsId='SearchBar__srInstructions'
+      headlessId='visual-autocomplete'
+      debounceTime={150}
+      renderVisualEntities={(isLoading) => (
+        <div className={isLoading ? 'opacity-50' : ''}>
+          <VisualEntities verticalKey='people'>
```

Comment:
I'm not sure what's a better name here: `VisualEntities` or `EntityPreviews`. I'd say lets get Product's opinion.